packages/legacy-client/src/client/objects/convertWireToOsdkObject.ts

```diff
@@ -60,13 +61,28 @@ function createPrototype<
     const self = this as OsdkLegacyPropertiesFrom<O, T> & OntologyObject<T>;
     for (const prop of Object.keys(objDef.properties)) {
       obj[prop] = self[prop];
+
+      if (isReservedKeyword(prop)) {
+        obj[`${prop}_`] = self[prop];
+      }
```

Comment:
if we set all the enumerable properties correctly, do we need to copy the object to JSON.stringify?

Reply:
You could maybe get away with that? But this matches the implementation from before where it copied them explicitly so I've been just adding from there.